pick.lic

```diff
@@ -463,7 +472,8 @@ class LockPicker
       harvest(box)
     when 'Roundtime'
       waitrt?
-      dispose_trash(left_hand) # only dispose https://elanthipedia.play.net/Locksmithing_skill#Box_Traps items.
+	  bput("put my #{left_hand} in my #{@component_container}", 'You put your', "You just can\'t get", "There isn\'t any more") if (@component_container != nil && left_hand =~ patterns)
```

Comment:
I think you can just do 

`if @component_container && left_hand =~ patterns`

or 

`if @component_container && (left_hand =~ patterns)`

Also, there's the .nil? method. `if !@component_container.nil?`